packages/design-system/src/tokens/ods/size.yaml

```
@@ -22,3 +22,8 @@ size:
     value: 22px
   form-check-default:
     value: 14px
+  tiles:
+    resizeable:
```

```suggestion
    resize-step:
```
or something similar? resizable (typo btw) sounds like a boolean.

---

Great idea, and thanks for the educate - turns out past me already made this mistake :D https://github.com/owncloud/web/pull/5052